staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -375,6 +375,11 @@ type CronJobSpec struct {
 	// The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
 	Schedule string `json:"schedule" protobuf:"bytes,1,opt,name=schedule"`
 
+	// The time zone for the given schedule, see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones.
+	// If not specified, this will rely on the time zone of the kube-controller-manager process.
+	// +optional
+	TimeZone *string `json:"timeZone,omitempty" protobuf:"bytes,8,opt,name=timeZone"`
```

Comment:
set this field to nil in `cronJobStrategy#PrepareForCreate` and `cronJobStrategy#PrepareForUpdate` if the feature is disabled and the field is not set in the old object (in PrepareForUpdate). See other strategy implementations for examples

Reply:
this is still outstanding